### PR TITLE
_fields: Allow url's to be 2000 characters.

### PR DESCRIPTION
**Pull request by edanaher**
Reference for the arbitary number:
https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Also, note that this submodule needs to be updated in neolith after this is merged.

### CODE REVIEW

File: lib/_fields.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ var fieldDefs = {
   url: {
     type: String,
     regEx: SimpleSchema.RegEx.Url,
-    max: 200
+    max: 2000
```
Comment (djrochford) [inline lib/_fields.js:144]: My only ask is that you leave that stack-overflow link in a comment somewhere near here.

Comment (edanaher) [reply lib/_fields.js:144]: Good call.

File: lib/_fields.js [hunk 2]
```diff
@@ -137,7 +137,11 @@ var fieldDefs = {
   url: {
     type: String,
     regEx: SimpleSchema.RegEx.Url,
-    max: 200
+    /* In AH-802 we had an uexpected error due to the 200 character limit.
```
Comment (djrochford) [inline lib/_fields.js]: "unexpected". Sorry!

Comment (edanaher) [reply lib/_fields.js]: I'm trying to be consistent with [umount](https://unix.stackexchange.com/questions/9832/why-is-umount-not-spelled-unmount)

Comment (edanaher) [reply lib/_fields.js]: (No I'm not.  Good catch.)

**Review by djrochford — APPROVED** [review]

**Review by fjlanasa — APPROVED** [review]

**Pull request merged**